app/controllers/SubscriptionController.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+use Webhooks\Models\Subscription;
+use Webhooks\Models\Event;
+use Webhooks\Models\Url;
+use Webhooks\Models\User;
+
+class SubscriptionController extends \BaseController {
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return Response
+     */
+    public function index()
+    {
+        $userId = 1;
+        $Event = Event::all();
+        return View::make("pages.subscribe",["Event"=>$Event]);
+    }
+
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return Response
+     */
+    public function eventActive()
+    {
+        $userId = 1;
+        Subscription::where('event_id',Input::get('eventId'))->where('user_id',$userId)->update(['active'=>Input::get('active')]);
```

Comment:
User subscriptions for an event x will be more readable than subscriptions for an event x and user y